Capstone 1 : Umuhanzi concert website.

BertrandConxy opened the pull request:
## Umuhanzi concert website
in this branch I have done the following:
- Built the HTML for the home page and about page.
- Added CSS styles to both pages. I like that I was able to use the sass preprocessor
- I used bootstrap too for layout and responsiveness.
- I created a featured artists section on the home page dynamically using javascript.
- I added menu button functionality using javascript.
- I made this mobile-first website responsive to large screens.
- After, I reordered a video for the presentation of my project. I recorded it using loom and the link is there below.
- https://www.loom.com/share/191c6b2b80764a0f978ea1c9d87292bd

BertrandConxy committed:
setting up all the neccesary prerequisites

BertrandConxy committed:
setting up all the neccessary prerequisites

BertrandConxy committed:
made the mobile skeleton of the website

BertrandConxy committed:
Added contents in home page and about page

BertrandConxy committed:
Add styles to showcase section

BertrandConxy committed:
Added styles to the schedule section and featured artists section

BertrandConxy committed:
correct linter errors

BertrandConxy committed:
added styles to the footer and about page

BertrandConxy committed:
added styles to the last section of the about page and corrected the view more button in homepage

BertrandConxy committed:
Added sytles to make the pages responsive

BertrandConxy committed:
completed syling the desktop version of the home-page

BertrandConxy committed:
stylesd about page

BertrandConxy committed:
created featured artists section dynamically, added toggle menu functionality, corrected linter errors

BertrandConxy committed:
Corrected linter errors

BertrandConxy committed:
made logo to link to the home page

BertrandConxy committed:
corrected the way to referrence the images

BertrandConxy committed:
Corrected the way I refference the images

BertrandConxy committed:
fixed the images not appearing in github pages

BertrandConxy committed:
corrected linter errors

BertrandConxy committed:
made some images to appear on github pages

BertrandConxy committed:
corrected image which was not visible in about page

BertrandConxy committed:
corrected images appearance

BertrandConxy committed:
corrected the images not appearing

BertrandConxy committed:
corrected angle down icon not appearing on github page

mmsesay commented on index.html:
```diff
@@ -0,0 +1,136 @@
+<header>
+    <i class="material-icons menu-btn">menu</i>
+</header>
+<nav class="navbar navbar-expand-sm bg-body navbar-light  shadow p-2 mb-5 rounded align-content-start  ">
+     
+    <div class="container-fluid  justify-content-start px-5">
+        <div class="container w-50 d-none d-lg-flex">
+    <a href="index.html" class=" navbar-brand logo btn"><i class="material-icons md-orange">audiotrack</i>Umuhanzi</a>
+  </div>
+    <i class="material-icons close-btn">close</i>
+      <ul class="navbar-nav">
+        <li class="nav-item p-2">
+          <a class="nav-link text-dark" href="index.html">Home</a>
+        </li>
+        <li class="nav-item p-2">
+          <a class="nav-link text-dark" href="about.html">About</a>
+        </li>
+        <li class="nav-item p-2">
+          <a class="nav-link text-dark" href="#artists">Artists</a>
+        </li>
+        <li class="nav-item p-2">
+            <a class="nav-link btn btn-menu" target="_blank" href="#"><i class="fas fa-play-circle px-2"></i>Stream Live</a>
+          </li>
+        <li class="nav-item p-2">
+          <a class="nav-link btn btn-menu2" target="_blank" href="#">Buy/Book Ticket</a>
+        </li>
+      </ul>
+    </div>
+  
+  </nav>
```
- [ ] Good job working on this project. Your page header is looking good but it misaligns a bit from the guided design.
It will be good if you could please implement the dark header on top the white header

Your Design |
:---------------:|
![e1](https://user-images.githubusercontent.com/39523312/148261366-05909126-994a-40fa-b77b-9239cd80fbae.png) 

Template Design |
:----------------------:|
![e2](https://user-images.githubusercontent.com/39523312/148261625-72aa45e8-3dfc-4e61-82c8-4f6899a61db4.png)

mmsesay commented on about.html:
```diff
@@ -0,0 +1,121 @@
+           </p>
+          <div class="container ">
+            <div class="row d-flex justify-content-center">
+               <div class="col-sm-3 p-4 m-3 project-img project-img-1"><h3>Iwacu muzika festival</h3><span>Season 1 in 2018</span><div class="overlay"></div></div>
+               <div class="col-sm-3 p-4 m-3 project-img project-img-2"><h3>Kigali fiesta</h3><span>summer festival</span><div class="overlay"></div></div>
+               <div class="col-sm-3 p-4 m-3 project-img project-img-3"><h3>New Year festival</h3><span>yearly festival</span><div class="overlay"></div></div>
```
- [ ] As per the design, only 2 images with text content are given. You provided 3 in your design. Please remove one.

mmsesay commented on style.css:
```diff
@@ -0,0 +1,645 @@
+  color: #ec5242 !important;
+  padding: 0.9em;
+  font-size: 1rem;
+  font-family: "Lato", sans-serif;
```
- [ ] [OPTIONAL]  I noticed you are using the font Lato in other areas in your code.
It will be a good practice to follow the[ CSS DRY METHOD](https://vanseodesign.com/css/dry-principles/). It help a lot to reduce code duplication.

mmsesay reviewed (changes requested):
Hi @BertrandConxy ,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlight!
- Good PR ✔️
- Good commits ✔️
- Professional readme ✔️
- Project idea originality ✔️
- Code organization ✔️
- Styling ✔️
- Good coordination in your video presentation and clear description of your project. :x: 
- Design resembles template: :x:

# Required Changes ♻️ ⛔️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

These are relating to your video:
  - [ ] Making your video 5 minutes max. It is currently above 5 minutes.
  - [ ] Just give a brief analysis of the project. Your readme explains some already.
  - [ ] Show the responsiveness of your project.
  - [ ] Show some part of your code that you are proud of making.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

BertrandConxy committed:
created the top header above the navigation bar as suggested by reviewer, removed the third image in the see past event section and fixed linter errors

BertrandConxy commented:
Thank you @mmsesay  I have implemented the changes you requested.
I the recent commits I build the top header above the navigation bar in the desktop version and I also removed one image from the see our past events section.
The new presentation video link is there below:
https://www.loom.com/share/191c6b2b80764a0f978ea1c9d87292bd

BertrandConxy committed:
Update README.md

rustam-tolipov reviewed (approved):
## Hi @BertrandConxy  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

![image](https://i.pinimg.com/originals/77/0e/b0/770eb057449582c44672b2d2d1cc39b6.gif)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
